apps/server/src/modules/group/controller/group.controller.ts

```diff
@@ -23,11 +23,14 @@ export class GroupController {
 	public async findClassesForSchool(
 		@Query() pagination: PaginationParams,
 		@Query() sortingQuery: ClassSortParams,
-		@CurrentUser() currentUser: ICurrentUser
+		@Query() schoolYearQuery: ClassFilterParams,
```

Comment:
I would name it more general like filterParams. So we dont have to rename it in the next ticket